apps/web/vibes/soul/primitives/skeleton/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { clsx } from 'clsx';
+
+type Props = {
+  parentClassName?: string;
+  childClassName?: string;
+  rounded?: 'sm' | 'base' | 'md' | 'lg' | 'xl' | '2xl' | '3xl' | 'full';
+} & (
+  | { type: 'text'; characterCount: number; aspectRatio?: never }
+  | { type: 'image'; aspectRatio: '5:6' | '3:4' | '1:1'; characterCount?: never }
+  | { type?: 'box' | 'circle'; characterCount?: never; aspectRatio?: never }
```

Comment:
`type: 'circle'` and `rounded: full` feel overloaded to me. Do you need a `type: 'circle'`?